content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -15,6 +15,17 @@ This document serves both as a reference to the values and as a coordination poi
 
 ## Labels, annotations and taints used on API objects
 
+
+### apf.kubernetes.io/autoupdate-spec
+
+Type: Annotation
+
+Example: `apf.kubernetes.io/autoupdate-spec: true`
+
+Used on: [All APF configuration objects](/concepts/cluster-administration/flow-control/#defaults)
```

Comment:
There aren't many kinds in the APF group, please spell them out.

Reply:
Changed, please take a look